### DISCUSSION TITLE
Why reviving QKSM and not other QKSM fork for example Partisan SMS?

### DISCUSSION THREAD

**Discussion by inson1** [octoshrimpy/quik#33] [Q&A] [answered]
Maybe its even good question to Readme

**Comment by octoshrimpy** [comment]
I'm not sure I understand the question.

**Reply by inson1** [reply]
@octoshrimpy Partisan SMS is dead fork of QKSM. But has more features than QKSM, so why reviving QKSM and not Partisan SMS?

I wrote "reviving" incorrectly, maybe that is the reason, why didnt you understand?

**Reply by inson1** [reply]
you didnt even use 3.10 branch right?

**Reply by e-t-l** [reply]
@octoshrimpy Hey Marcos, I have a similar question (at least, I think so. The above is a little hard to parse. Let me see if I can phrase my question more clearly)

Do you have a particular reason why you started your own fork of QKSMS, as opposed to contributing to (or at least forking off of) an existing fork like [wrwrabbit/Partisan-SMS](https://github.com/wrwrabbit/Partisan-SMS)? Partisan has added some optional encryption features, but I *think* they have also made some general improvements to the app. Have you checked if they've done anything worth copying? (note: I'm actually not sure if Partisan is still actively maintained; I just now posted in their Discussions asking about it)

**Reply by inson1** [reply]
@e-t-l idk Why is the title hard to parse? Partisan is dead, 9 months is too long.

**Reply by octoshrimpy** [reply, marked as answer]
I've taken this undertaking as a way of becoming more comfrotable with CI/CD, as well as learn kotlin. I did use the 3.10 branch, and had it come back and bite me once already :sweat_smile: 

As of now, I do believe quik is the most up-to-date fork of QKSMS.
